mods/persistence/game/objects/items/items.dm

```diff
@@ -0,0 +1,16 @@
+SAVED_VAR(/obj/item, health)
+
+//Used by things that can be collapsed and put into other inv slots
+SAVED_VAR(/obj/item, slot_flags)
```

Comment:
It can wait, but this should really be set on specific subtypes that have this behavior. Might want to add a TODO

Reply:
Eh, I think people can figure it out. If we add todo everywhere people will even care less about them.